packages/electrode-bolt/package.json

```diff
@@ -1,22 +1,22 @@
 {
   "name": "electrode-bolt",
   "version": "1.0.9",
-  "description": "Opinionated meta config runner for react components",
+  "description": "Builder Archetype - Electrode Component",
   "scripts": {
-    "dev": "bolt server-dev & bolt server-test",
-    "hot": "bolt server-hot & bolt server-test",
+    "dev": "builder run server-dev & builder run server-test",
+    "hot": "builder run server-hot & builder run server-test",
     "lint-react-demo": "eslint --ext .js,.jsx -c ./node_modules/electrode-bolt/config/eslint/.eslintrc-react-demo demo/*.jsx --color",
     "lint-react-src": "eslint --ext .js,.jsx -c ./node_modules/electrode-bolt/config/eslint/.eslintrc-react src --color",
     "lint-react-test": "eslint --ext .js,.jsx -c ./node_modules/electrode-bolt/config/eslint/.eslintrc-react-test test/client --color",
-    "lint": "bolt lint-react-demo && bolt lint-react-src && bolt lint-react-test",
+    "lint": "builder run lint-react-demo && builder run lint-react-src && builder run lint-react-test",
```

Comment:
OPTIONAL: Could do `builder concurrent lint-react-demo lint-react-src lint-react-test` for speed.